src/command_modules/azure-cli-network/azure/cli/command_modules/network/custom.py

```diff
@@ -215,62 +220,63 @@ def set_lb_inbound_nat_pool(
         frontend_ip_name=None):
     ncf = _network_client_factory()
     lb = ncf.load_balancers.get(resource_group_name, load_balancer_name)
-    item = _get_lb_property(lb.inbound_nat_pools, item_name)
+    item = _get_property(lb.inbound_nat_pools, item_name)
+
     _set_param(item, 'protocol', protocol)
     _set_param(item, 'frontend_port_range_start', frontend_port_range_start)
     _set_param(item, 'frontend_port_range_end', frontend_port_range_end)
     _set_param(item, 'backend_port', backend_port)
-    if frontend_ip_name == "":
+
+    if frontend_ip_name == '':
         item.frontend_ip_configuration = None
     elif frontend_ip_name is not None:
         item.frontend_ip_configuration = \
-            _get_lb_property(lb.frontend_ip_configurations, frontend_ip_name)
+            _get_property(lb.frontend_ip_configurations, frontend_ip_name)
+
     return ncf.load_balancers.create_or_update(resource_group_name, load_balancer_name, lb)
 
 # Frontend IP Configuration
 
 def create_lb_frontend_ip_configuration(
-        resource_group_name, load_balancer_name, item_name, public_ip_address_name=None,
-        virtual_network_name=None, subnet_name=None, private_ip_address=None):
+        resource_group_name, load_balancer_name, item_name, public_ip_address=None,
+        subnet=None, virtual_network_name=None, private_ip_address=None, # pylint: disable=unused-argument
```

Comment:
No because then the aliasing wouldn't work. 

---
In reply to: [73073569](https://github.com/Azure/azure-cli/pull/592#discussion_r73073569) [](ancestors = 73073569)